modules/RTC/RTCUtils.js

```diff
@@ -31,7 +31,7 @@ const logger = getLogger(__filename);
 // Require adapter only for certain browsers. This is being done for
 // react-native, which has its own shims, and while browsers are being migrated
 // over to use adapter's shims.
-if (browser.usesNewGumFlow()) {
+if (browser.usesNewGumFlow() || browser.isEdge()) {
```

Comment:
But we use usesNewGumFlow in multiple places and adding Edge as usesNewGumFlow will change the behavior. 
The goal of this change is only to load the adapter when running on Edge without changing anything else. 
The goal is to use the iceServers filter function which is in the adapter, that's why I deleted our incomplete copy of it from our code.

Reply:
but does edge use the new gum flow or not?

Reply:
oh, i see now...sorry, i didn't read the surrounding code closely enough.  yeah this looks like it should be a new test instead of `usesNewGumFlow`, like `useAdapter`?

Reply:
I will add `usesAdapter` in BrowserCapabilities.js, does this look ok?
```
    usesAdapter() {
        return this.usesNewGumFlow() || this.isEdge();
    }
``` 
And the check becomes:
```
if (browser.usesAdapter()) {
    require('webrtc-adapter');
}
```
`uses` rather than `use` to follow the other method names.

Reply:
yeah sounds good